lib/these.js

```diff
@@ -83,23 +88,48 @@ These.prototype.map = function (f) {
     return new These(this._this, f(this._that));
 };
 
+This.prototype[FL.map] = This.prototype.map;
+That.prototype[FL.map] = That.prototype.map;
+These.prototype[FL.map] = These.prototype.map;
+
 // Apply : ap
 
 This.prototype.ap = function (x) { return x.map(this._this); };
 That.prototype.ap = function (x) { return x.map(this._that); };
 These.prototype.ap = function (x) { return x.map(this._that); };
```

Comment:
I'm surprised that these don't need to change, given that the type of the `ap` method changed in the spec. Were these methods compliant with `fantasy-land@0.3.x`?